ui/helpers/job.js

```diff
@@ -177,7 +177,7 @@ export const findGroupInstance = function findGroupInstance(job) {
 // Fetch the React instance based on the jobId, and if scrollTo
 // is true, then scroll it into view.
 export const findJobInstance = function findJobInstance(jobId, scrollTo) {
-  const jobEl = $('.th-view-content')
+  const jobEl = $('#push-list')
```

Comment:
Changed this to make it easier in the unit tests. It creates the same constraint to find the right elements.

Reply:
Can you remove the jquery in this pr or are you planning to remove it all it once in another one? :)

Reply:
Yeah, I'll remove JQuery usages in a follow-up PR>